src/fides/api/api/v1/api.py

```diff
@@ -29,7 +29,7 @@
 api_router.include_router(connection_type_endpoints.router)
 api_router.include_router(connection_endpoints.router)
 api_router.include_router(consent_request_endpoints.router)
-api_router.include_router(dataset_endpoints.router)
+api_router.include_router(dataset_config_endpoints.router)
```

Comment:
Renaming this to `dataset_config_endpoints` since that is more accurate. You'll notice a more consistent separation between connection datasets (DatasetConfigs) and standalone datasets in the rest of the changes